packages/flutter/test/material/page_test.dart

```diff
@@ -72,7 +73,7 @@ void main() {
 
     expect(find.text('Page 1'), isOnstage);
     expect(find.text('Page 2'), findsNothing);
-  }, variant: TargetPlatformVariant.only(TargetPlatform.android));
+  }, variant: TargetPlatformVariant.only(TargetPlatform.android), skip: kIsWeb); // [intended] Transitions disabled on web
```

Comment:
I took all of these skips from the diff that was accepted in #82596 (cc @darrenaustin as an FYI)